### ISSUE TITLE
Moving a terminal editor tab goes blank

### ISSUE THREAD

**Issue by s3than** [microsoft/vscode#138797]
Issue Type: <b>Bug</b>

When moving the terminal to a new split it goes blank. Opening a new tab in that split will work..

VS Code version: Code 1.63.0 (7db1a2b88f7557e0a43fec75b6ba7e50b3e9f77e, 2021-12-07T05:15:48.091Z)
OS version: Darwin x64 20.6.0
Restricted Mode: No
Remote OS version: Linux x64 5.13.0-20-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-1068NG7 CPU @ 2.30GHz (8 x 2300)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|5, 3, 3|
|Memory (System)|32.00GB (7.22GB free)|
|Process Argv|--crash-reporter-id 361698e9-7f01-4409-9679-4dcd302c31c7|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: Dev|
|OS|Linux x64 5.13.0-20-generic|
|CPUs|Intel(R) Core(TM) i7-1068NG7 CPU @ 2.30GHz (4 x 2300)|
|Memory (System)|7.77GB (6.32GB free)|
|VM|0%|
</details><details><summary>Extensions (21)</summary>

Extension|Author (truncated)|Version
---|---|---
Nix|bbe|1.0.1
gotemplate|cas|0.4.0
jupyter-keymap|ms-|1.0.0
remote-containers|ms-|0.209.3
remote-ssh|ms-|0.68.0
remote-ssh-edit|ms-|0.68.0
vscode-icons|vsc|11.8.0
HCL|who|0.0.5
vscode-markdownlint|Dav|0.45.0
prettier-vscode|esb|9.0.0
vscode-diff|fab|1.4.2
go|gol|0.29.0
terraform|has|2.17.0
rest-client|hum|0.24.6
vscode-kubernetes-tools|ms-|1.3.4
indent-rainbow|ode|8.2.2
vscode-yaml|red|1.2.2
code-spell-checker|str|2.0.13
gitblame|wad|8.1.0
highlight-trailing-white-spaces|yba|0.0.2
markdown-all-in-one|yzh|3.4.0

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551:30345470
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263cf:30335440
vscoreces:30384385
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsaa593:30376534
pythonvs932:30410667
vscop804cf:30404767
vs360cf:30404996
vsrem710cf:30405999
vscexrecpromp3t2:30407763

```

</details>

**Comment by shaperilio** [comment]
And moving it back to where you created it will make it work again.

This most definitely did not happen in the previous release; I just tried this before and after the update.

**Comment by Tyriar** [comment]
Might be seeing https://github.com/microsoft/vscode/issues/138452

Can you upload a [gif](http://gifcap.dev/) showing the problem happen?

**Comment by shaperilio** [comment]
Just confirmed it works fine in the insiders' build:

Version: 1.64.0-insider (user setup)
Commit: 065b450db9f555f177161748e8f2911ebb5677f9
Date: 2021-12-10T05:14:44.789Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043

**Comment by shaperilio** [comment]
![2d1bca74-c658-4e90-ae86-ef2d3fda7836](https://user-images.githubusercontent.com/26465923/145604654-95422b56-e51e-48a3-b20d-1a7021289bf3.gif)

Version: 1.63.0 (user setup)
Commit: 7db1a2b88f7557e0a43fec75b6ba7e50b3e9f77e
Date: 2021-12-07T06:26:56.179Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043

This is recorded in the window that launches when I start VSCode from the start menu. I then created a new window (CTRL+SHIFT+N), created a new file **before** opening the terminal, and then I was able to move the terminal just fine. Repeating that back in the main window did not work - so it's not 100% repeatable.
